src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$calendar-padding: 10px;
+$cell-color: #eee;
+$cell-color-hover: #ffc0cb;
+
+$daysOfWeek:
+  "mon" 0,
+  "tue" $cell-size + $cell-gap,
+  "wed" 2 * $cell-size + 2 * $cell-gap,
+  "thu" 3 * $cell-size + 3 * $cell-gap,
+  "fri" 4 * $cell-size + 4 * $cell-gap,
+  "sat" 5 * $cell-size + 5 * $cell-gap,
+  "sun" 6 * $cell-size;

```

Comment:
I think you can create a function to calculate margin

Reply:
done